projects/schematics/src/shared/utils/file-utils.ts

```diff
@@ -122,37 +122,25 @@ export function insertHtmlComment(
   componentSelector: string,
   componentProperty: ComponentProperty
 ): string | undefined {
-  const selector = buildSelector(componentSelector);
+  let newContent = content;
```

Comment:
We should add tests that cover the `*ngIf` cases

Reply:
just added it